pkg/queue/manager.go

```diff
@@ -206,6 +206,16 @@ func (m *Manager) DeleteClusterQueue(cq *kueue.ClusterQueue) {
 	metrics.ClearClusterQueueMetrics(cq.Name)
 }
 
+func (m *Manager) DefaultLocalQueue(namespace string) string {
```

Comment:
Since we always check the result of this function in if statement and compare it with `""`, can this function return `true` if there is the default LocalQueue in the namespace and `false` otherwise. If so, then can we rename it to something along the lines `IsDefaultLQInNamespace`

Reply:
Done